### ISSUE TITLE
Problems with fullscreen

### ISSUE THREAD

**Issue by boasbakker** [xCollateral/VulkanMod#398]
Hardware/software: Same as previous, no other mods.

[01:03:03] [Render thread/ERROR]: Not supported resolution, fallback to first supported
[01:04:22] [Render thread/ERROR]: ########## GL ERROR ##########
[01:04:22] [Render thread/ERROR]: @ Render
[01:04:22] [Render thread/ERROR]: 65544: Win32: Failed to set video mode: Graphics mode not supported
[01:05:10] [Render thread/ERROR]: ########## GL ERROR ##########
[01:05:10] [Render thread/ERROR]: @ Render
[01:05:10] [Render thread/ERROR]: 65544: Win32: Failed to set video mode: Graphics mode not supported

I get these errors when using fullscreen mode / F11. Every time I switch to/from fullscreen the whole monitor becomes black for a few seconds. When using fullscreen the screen is skewed and the color is a bit white, but both don't show on a screen recording with OBS. When trying to take a screenshot with Windows+Shift+S the black screen appears again and Minecraft is minimized. https://youtu.be/vcXF4oE4OYw this is the recording. Windows fullscreen works perfectly.

**Comment by boasbakker** [comment]
![image](https://github.com/xCollateral/VulkanMod/assets/59883944/aa058f06-694f-47c7-83ec-94da1bea15e5)
It seems some resolutions work better than others. This one worked okay on my 1440x900 primary (very old) monitor after switching to another app and back (still black screen though). The white and stretching seemed to be gone or at least less noticeable and I could take this screenshot with Windows+Shift+S. Still errors in the logs though

**Comment by boasbakker** [comment]
I also encountered other glitches like the mouse clicking on the wrong items and not getting out of fullscreen (even with the settings) anymore.

**Comment by Logboy2000** [comment]
Might be worth opening another issue for this but heres another problem in fullscreen. 

When i have 2 monitors connected the game will show on both with one being the actual game and the other being a frozen frame

**Comment by Versatile-V1** [comment]
@boasbakker Is this still an issue?

**Comment by boasbakker** [comment]
> [@boasbakker](https://github.com/boasbakker) Is this still an issue?

I can test this in a few days

**Comment by boasbakker** [comment]
> [@boasbakker](https://github.com/boasbakker) Is this still an issue?

Yep this still happens on 1.21.10 with the latest release of VulcanMod

**Comment by Versatile-One** [comment]
> > [@boasbakker](https://github.com/boasbakker) Is this still an issue?
> 
> Yep this still happens on 1.21.10 with the latest release of VulcanMod

What about the latest version now? If it doesn't happen then this can be closed

**Comment by boasbakker** [comment]
> > > [@boasbakker](https://github.com/boasbakker) Is this still an issue?
> > 
> > 
> > Yep this still happens on 1.21.10 with the latest release of VulcanMod
> 
> What about the latest version now? If it doesn't happen then this van be closed

I'm not able to test this any time soon, maybe someone else with similar hardware can test it?
